x-pack/plugins/ingest_manager/server/services/epm/elasticsearch/transform/remove.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { SavedObjectsClientContract } from 'kibana/server';
+import { CallESAsCurrentUser, ElasticsearchAssetType, EsAssetReference } from '../../../../types';
+import { PACKAGES_SAVED_OBJECT_TYPE } from '../../../../../common/constants';
+
+export const stopTransforms = async (transformIds: string[], callCluster: CallESAsCurrentUser) => {
+  for (const transformId of transformIds) {
+    await callCluster('transport.request', {
+      method: 'POST',
+      path: `/_transform/${transformId}/_stop`,
+      query: 'force=true',
+      ignore: [404],
+    });
+  }
+};
+
+export const deleteTransforms = async (
+  callCluster: CallESAsCurrentUser,
+  transformIds: string[]
+) => {
+  for (const transformId of transformIds) {
+    await stopTransforms([transformId], callCluster);
+    await callCluster('transport.request', {
+      method: 'DELETE',
+      query: 'force=true',
+      path: `_transform/${transformId}`,
+      ignore: [404],
```

Comment:
In the above delete command, you ignore 404 and 400. What is the difference?

Reply:
Removed all the 400, it is a vestige of my initial testing.